src/api-service/__app__/onefuzzlib/events.py

```diff
@@ -33,9 +35,60 @@ def get_events() -> Optional[str]:
         return None
 
 
+def log_event(event: Event, event_type: EventType) -> None:
```

Comment:
is this used ?

Reply:
I think it was meant to be used in send_event below instead of filter_event

Reply:
You were correct.